src/Nethermind/Nethermind.Runner/configs/kovan.cfg

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
do we have config tests for kovan.cfg? there is an extensive config tester in the Nethermind.Runner project